opencodelists/tests/views/test_user.py

```diff
@@ -0,0 +1,118 @@
+from datetime import datetime
+
+from builder.actions import save as save_for_review
+from codelists.actions import (
+    create_codelist_from_scratch,
+    publish_version,
+    update_codelist,
+)
+from opencodelists.actions import add_user_to_organisation
+from opencodelists.models import User
+
+
+def test_get(client, organisation_user):
+    user = User.objects.create(
+        username="test", name="test", email="test@test.com", password="test"
+    )
```

Comment:
I'd prefer it if we created a `user_with_no_codelists` fixture, instead of creating it here.